README.md

```diff
@@ -16,15 +16,19 @@ Supported engines:
 
 ## Build and Run
 ### Linux
+Dependencies:
```

Comment:
I think do not need the Linux/OSX separation anymore, you can simply remove the OSX part. Remove "(OpenJDK recommended)".

Reply:
(java noob) : Seems that OS X is using different commands and it's usually a good idea to separate OSX and Linux since they are very different i believe

Reply:
Ah i see what you mean readme points to https://phoenicisorg.github.io/phoenicis/ now.. aldo i'm unable to find dependencies on it?

Reply:
I will do the update.